### PR TITLE
Use response.text and response.data

### PR DESCRIPTION
**Pull request by MarkDaoust**
Use response.text and response.data

Also:

 live_api_starter.ipynb
 
 - Use `enumerate` instead of `first` 
 - Give option to pass `turns=` when creating the autio loop so it can run without interaction.
 
live_api_starter.py

- Use a single output message queue
- Cleanup the exception handling
-  fix BGR2RGB ( like https://github.com/google-gemini/cookbook/pull/338 )
- Limit output queue size
- `exception_on_overflow=False` when debugging so it doesn't crash as sook as you pause.

websockets/live_api_starter.py

- Use a single oputput message queue
- Cleanup the exception handling

README.md 

- Mention video out.

### CODE REVIEW

**Comment by MarkDaoust** [comment]
<p>Any suggestions? async_enumerate?</p><p> </p>

---

 <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/344/discussion/#comment-2539167745'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by markmcd — DISMISSED** [review]
Awesome!

**Review by Giom-V — DISMISSED** [review]

**Comment by Giom-V** [comment]
<p>I think it would be clearer yes.</p>

---

 <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/344/discussion/#comment-2539167745'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: gemini-2/live_api_starter.py [hunk 1]
```diff
@@ -27,42 +26,36 @@
 from google import genai
 
 if sys.version_info < (3, 11, 0):
-    import taskgroup, exceptiongroup
-    asyncio.TaskGroup = taskgroup.TaskGroup
-    asyncio.ExceptionGroup = exceptiongroup.ExceptionGroup
+    raise EnvironmentError("This script requires python 3.11+")
```
Comment (MarkDaoust) [inline gemini-2/live_api_starter.py:29]: Maybe revert this depending on if https://github.com/google-gemini/cookbook/pull/353 solves it.

Comment (MarkDaoust) [reply gemini-2/live_api_starter.py]: Done.

**Review by Giom-V — APPROVED** [review]

**Review by iqbalbhatti49 — COMMENTED** [review]
Thanks

**Pull request merged**